frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -117,6 +118,7 @@ class BundleRow extends Component {
             focusIndex,
             ws,
         } = this.props;
+        const stateInfoFromWorksheet = getStateInfo(bundleInfo);
```

Comment:
Would it make more sense to pass the `bundleInfo` directly down into `BundleDetail`?  `stateInfo` right now is a helper that mixes in rendering logic.  I think perhaps we want to keep things more semantic and let `BundleDetail` deal with how it wants to render the information.  This way, we also don't have another type `StateInfo` to reason about.